control_panel_api/views.py

```diff
@@ -58,6 +58,10 @@ class AppS3BucketViewSet(viewsets.ModelViewSet):
     queryset = AppS3Bucket.objects.all()
     serializer_class = AppS3BucketSerializer
 
+    def perform_destroy(self, instance):
```

Comment:
why do all these method names start with `perform_`? isn't that redundant?

Reply:
This is not exactly apparent and I may be wrong but I think here we're overriding a method coming from Django Rest Framework (the beauty of inheritance 💃 )

Reply:
@andyhd what @xoen said!